_transactions.md

```diff
@@ -267,6 +267,140 @@ url        | The URL of the 3DSecure confirmation request.
 
 A webpage for 3DSecure confirmation for the user to interact with.
 
+## Beneficiary Information
+
+> Example of a transaction creation payload including `beneficiary` and `purpose` fields:
+
+```json
+{
+  "beneficiary": {
+    "relationship": "child",
+    "name": "Harper Wilson",
+    "address": {
+      "line1": "Bar",
+      "zipCode": "12345",
+      "city": "Braga",
+      "country": "PT",
+      "state": "PT-03"
+    }
+  },
+  "purpose": "donations",
+  "denomination": {
+    "amount": "3000",
+    "currency": "USD"
+  },
+  "destination": "invite-user@gmail.com"
+}
+```
+
+When creating a transaction, it may be required to send additional information
+regarding the beneficiary and purpose of the transaction.
+This is required if **both** the following conditions are met:
+
+- The transaction is either a withdrawal to an external account (crypto networks, SEPA bank accounts, etc.) or a transfer to another user; and
+- The transaction amount is over _$3000 USD_ (or over $1000 USD, if the origin user is from Arizona).
+
+<aside class="notice">
+  The threshold value is calculated from the <a href="#normalized">normalized</a> amount of the transaction,
+  which can differ from the amount specified in the <code>denomination.amount</code> field
+  if the transaction implies a currency conversion.
+</aside>
+
+The beneficiary information should be provided in the `beneficiary` object of the request payload.
+The `beneficiary.relationship` field must always be filled whenever beneficiary information is required.
+The other fields can be omitted if the data is already present in the Uphold platform
+(e.g. if the transaction is to an existing Uphold user, or the `beneficiary.relationship` is set to "myself").
+
+The `relationship` field is a string reflecting the beneficiary's relationship to the transaction originator.
+Two particular types of transactions are recognized by the platform:
+business transactions (indicated by a `relationship` field with the value `business`)
+and personal transactions (indicated by the values `child`, `co_worker`, `friend`, `parent`, or `sibling`).
+The special value `myself` can also be used when applicable.
+
+Depending on the conditions of the transaction, additional fields may be required
```

Comment:
Is this newline with a dash intentional?

Reply:
It is intentional. I've been making use of [semantic line breaks](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) in the docs to facilitate reading of the the source files and make diffs more concise and readable, though I haven't applied that consistently throughout the repo yet. I can remove this particular instance if it's jarring to you.

Reply:
Oh, that's genious. Great concept ❤️ 

Anyway, I think what threw me off was the dash (`-`) itself. Maybe we can just remove it?

Reply:
Yeah, there is [some discussion](https://github.com/sembr/specification/issues/4#issuecomment-835668183) about whether/how to break on dashes — it's certainly not a clear-cut case. I'll go ahead and remove this break.

Reply:
[Done](https://github.com/uphold/docs/compare/34ed6078fff1f23bc041e71599d7b0c41bde2604..98edf50ba5a5c1de6193428a66f626b5463b40e3).